includes/definitions/discovery/hwg-ste2.yaml

```diff
@@ -1,7 +1,7 @@
 mib: STE2-MIB
 modules:
     os:
-        sysDescr_regex: '/(?<hardware>.+?)( (?<version>[\d.]+))?/'
+        sysDescr_regex: '/(?<hardware>.*),\s+fw:(?<version>.*)?/'
```

Comment:
not sure of this regexp (final '?') ?

You may try this : 
`sysDescr_regex: '/(?<hardware>[^,]*),\s+fw:(?<version>.*)/'`

Everything up to the "," is hardware, everything after "fw:" is version

Reply:
at least it seems to work as expected.